libs/crypto/certificate-generation/src/main/kotlin/net/corda/crypto/test/certificates/generation/CertificateAuthorityFactory.kt

```
@@ -40,4 +42,31 @@ object CertificateAuthorityFactory {
             keysFactoryDefinitions, validDuration, home
         )
     }
+
+    /**
+     * Create a local certificates authority that support revocation of certificate.
+     *
+     * @param keysFactoryDefinitions - The keys factory definitions (currently, only RSA and EC are supported)
+     * @param validDuration - The duration after which the certificate will become invalid
+     * @param host - The revocation server host name (default to localhost).
+     * @param port - The revocation server port number (default to a random open port).
```

Does the port defaults to a random open port?